Makefile

```diff
@@ -38,7 +38,9 @@ GOOS ?= $(shell go env GOOS)
 
 NO_DOCKER ?= 0
 
-ifeq ($(shell command -v podman > /dev/null 2>&1 ; echo $$? ), 0)
+ifeq (${IS_CONTAINER}, TRUE)
```

Comment:
FYI this is what we did in 4.14/4.13 re this issue: https://github.com/openshift/machine-api-provider-azure/pull/74/files

Reply:
Right, that would have worked but does change the dev experience. Given this is 4.12, I don't think I care much about consistency for this, would rather just get it fixed. The net result is the same 😅